src/main/java/net/javadiscord/javabot/systems/qotw/submissions/SubmissionManager.java

```diff
@@ -57,28 +65,25 @@ public class SubmissionManager {
 	public WebhookMessageCreateAction<?> handleSubmission(@NotNull ButtonInteractionEvent event, int questionNumber) {
 		event.deferEdit().queue();
 		Member member = event.getMember();
-		if (!this.canCreateSubmissions(member)) {
+		if (!canCreateSubmissions(member)) {
 			return Responses.warning(event.getHook(), "You're not eligible to create a new submission thread.");
 		}
 		config.getSubmissionChannel().createThreadChannel(
-				String.format(THREAD_NAME, questionNumber, member.getEffectiveName()), true).queue(
+				String.format(THREAD_NAME, questionNumber, member.getId()), true).queue(
 				thread -> {
 					thread.addThreadMember(member).queue();
 					thread.getManager().setInvitable(false).setAutoArchiveDuration(ThreadChannel.AutoArchiveDuration.TIME_1_WEEK).queue();
 					try {
```

Comment:
What's the point of this `try`-`catch` block if everything in it is executed asynchronously?

Reply:
fixed